### PR TITLE
feat: remove project directory + improve docs

### PR DESCRIPTION
**Pull request by ZeRego**
### Pre requirements:
- [x] I have followed (at least) the [PR section of the contributing guide](https://github.com/lightdash/lightdash/blob/main/.github/CONTRIBUTING.md#sending-a-pull-request).

### Reference:
Closes: #1065 and #1186

### Preview:
Screenshot 2022-01-21 at 10 14 50

### Scope of the changes (select all that apply):
- [x] Frontend  
- [ ] Backend  
- [ ] Common  
- [ ] E2E tests  
- [x] Docs  
- [ ] CI/CD

### CODE REVIEW

File: docs/docs/get-started/setup-lightdash/install-lightdash.mdx [hunk 1]
```diff
@@ -60,9 +60,8 @@ cd lightdash
 
 :::info
 
-If you're using a local dbt project, when you're asked `how you want to setup Lightdash`, select the option **"with local dbt"**
-and during the project creation keep the **Project directory** value as **"/usr/app/dbt"**.
-
+If you are using the install script, when you are asked **how do you want to setup Lightdash ?**, select the option  **with local dbt** and then provide the absolute path to your dbt project.
```
Comment (hamzahc1) [inline docs/docs/get-started/setup-lightdash/install-lightdash.mdx:63]: Suggestion:

When using the install script, when you're asked `how do you want to setup Lightdash ?`, select the option  **with local dbt** and then provide the absolute path to your dbt project.

File: docs/docs/get-started/setup-lightdash/install-lightdash.mdx [hunk 2]
```diff
@@ -60,9 +60,8 @@ cd lightdash
...
+If you are installing manually, set the env var **DBT_PROJECT_DIR** with the absolute path to your dbt project.
```
Comment (hamzahc1) [inline docs/docs/get-started/setup-lightdash/install-lightdash.mdx:64]: Suggestion:

When installing manually, set the env var `DBT_PROJECT_DIR` with the absolute path to your dbt project.

File: packages/frontend/src/components/ProjectConnection/DbtForms/DbtLocalForm.tsx [hunk 3]
```diff
@@ -1,13 +1,24 @@
...
+const DbtLocalForm: FC = () => (
     <>
+        <Callout intent="warning" style={{ marginBottom: 20 }}>
+            This connection type should only be used for local development.
+        </Callout>
         <Callout intent="primary" style={{ marginBottom: 20 }}>
-            This connection type should only be used for local development. Read
-            docs{' '}
+            <p>
+                If you are using the install script, when you are asked{' '}
```
Comment (hamzahc1) [inline packages/frontend/src/components/ProjectConnection/DbtForms/DbtLocalForm.tsx:11]: Same sentences as above ^

File: docs/docs/get-started/setup-lightdash/install-lightdash.mdx (hunk 2 above)
Comment (ZeRego) [reply docs/docs/get-started/setup-lightdash/install-lightdash.mdx:64]: Changes in a new PR https://github.com/lightdash/lightdash/pull/1213

**Pull request merged**